src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
```

Comment:
it's better to always style by classes, not by tags (except for body and html tags)